assets/base/config.json

```diff
@@ -12,7 +12,6 @@
     "MobileNoticeURL": "https://about.mattermost.com/mobile-notice-txt/",
     "SegmentApiKey": "oPPT2MA24VYZTBGDyXtUDa7u50lkPIE4",
     "ExperimentalUsernamePressIsMention": false,
-    "ExperimentalClientSideCertEnable": false,
```

Comment:
Is this on for everyone now?

Reply:
yes it is, it will depend on the nginx config